_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-filter-item.vue

```diff
@@ -147,6 +148,12 @@ export default {
       fillTableOfChildren(item);
       return tableOfCHildren.length;
     },
+    idAndName(item) {
+      if (this.$store.state.smartShoppingCampaigns.dimensionChosen.id === 'conditions') {
+        return item.id !== item.name ? `${item.id} - ${item.name}` : this.$t(`smartShoppingCampaignCreation.productConditions.${item.name}`);
```

Comment:
I believe we can specify this only rule for condition :+1: 

```suggestion
        return this.$t(`smartShoppingCampaignCreation.productConditions.${item.name}`);
```

Reply:
But then what happened in brands since we don't have the translations

Reply:
As you made clear in the `if` that you only cover conditions here, Brands will fall on the second return